es/channel/internal.js

```diff
@@ -27,31 +26,24 @@ const PING_TIMEOUT_MS = 10000
 // Close connection if pong message is not received within 5 seconds
 const PONG_TIMEOUT_MS = 5000
 
-const options = new WeakMap()
-const status = new WeakMap()
-const state = new WeakMap()
+export const options = new WeakMap()
+export const status = new WeakMap()
+export const state = new WeakMap()
 const fsm = new WeakMap()
 const websockets = new WeakMap()
-const eventEmitters = new WeakMap()
+export const eventEmitters = new WeakMap()
 const messageQueue = new WeakMap()
 const messageQueueLocked = new WeakMap()
 const actionQueue = new WeakMap()
 const actionQueueLocked = new WeakMap()
 const sequence = new WeakMap()
-const channelId = new WeakMap()
+export const channelId = new WeakMap()
 const rpcCallbacks = new WeakMap()
 const pingTimeoutId = new WeakMap()
 const pongTimeoutId = new WeakMap()
-const fsmId = new WeakMap()
+export const fsmId = new WeakMap()
 
-function channelURL (url, params) {
-  const paramString = R.join('&', R.values(R.mapObjIndexed((value, key) =>
-    `${pascalToSnake(key)}=${encodeURIComponent(value)}`, params)))
-
-  return `${url}?${paramString}`
-}
```

Comment:
Isn't this used anywhere?

Reply:
It is used only in this file, in `initialize` function, otherwise, it will be rejected by linter.